tests/integration-tests/tests/dcv/test_dcv.py

```diff
@@ -27,7 +27,7 @@
 @pytest.mark.parametrize(
     "dcv_port, access_from, shared_dir", [(8443, "0.0.0.0/0", "/shared"), (5678, "192.168.1.1/32", "/myshared")]
 )
-@pytest.mark.regions(["eu-west-1", "cn-northwest-1"])  # DCV license bucket not present in us-gov
+@pytest.mark.regions(["us-east-2", "cn-northwest-1"])  # DCV license bucket not present in us-gov
```

Comment:
just curious: why did this need to be changed?

Reply:
Just because my AMI is in `us-east-2`. Generally this could be any commercial region, but I will change in back just for consistency.